src/main/java/seedu/planner/logic/commands/AddCommand.java

```diff
@@ -52,47 +61,91 @@ public AddCommand(Set<Module> modules, int index) {
     }
 
     @Override
-    public CommandResult execute(Model model, CommandHistory history) throws CommandException {
+    public CommandResult execute(Model model, CommandHistory history) {
```

Comment:
Would be good to extract out some code into smaller methods. This method seems pretty large